Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -462,6 +472,33 @@ function Add-RegistrationApiRegistration([string] $identityServerUrl, [string] $
     return $registrationApiSecret
 }
 
+function Add-IdpssApiResourceRegistration($identityServiceUrl, $fabricInstallerSecret)
+{
+    $accessToken = Get-AccessToken -authUrl $identityServiceUrl -clientId "fabric-installer" -secret $fabricInstallerSecret -scope fabric/identity.manageresources
+
+    Write-DosMessage -Level Information "Registering IdentitySearchProvider API with Fabric.Identity..."
+    $apiName = "identitySearchProvider-service-api"
```

Comment:
I think today we register this with this name `idpsearch-api`

Reply:
ok